client/web/src/enterprise/batches/detail/BatchChangeDetailsTabs.tsx

```diff
@@ -268,6 +268,16 @@ export const BatchChangeDetailsTabs: React.FunctionComponent<React.PropsWithChil
                     )}
                 </TabPanel>
                 <TabPanel>
+                    <Text className="my-3">
+                        Archived changesets are changesets created and published by an earlier version of the batch spec
```

Comment:
```suggestion
                        Archived changesets are changesets created and published by an earlier version of the batch change
```

I think this is slightly more accurate.